### PR TITLE
Translation Added for Column Pattern

### PR DESCRIPTION
**Pull request by devmuhib009**
Added translation version of pattern column.

### CODE REVIEW

File: patterns/columns.php [hunk 1]
```diff
@@ -12,51 +12,51 @@
 <div class="wp-block-group alignfull has-base-background-color has-background" style="padding-top:7em;padding-right:7em;padding-bottom:7em;padding-left:7em"><!-- wp:columns {"align":"wide"} -->
 <div class="wp-block-columns alignwide"><!-- wp:column {"width":"33%","style":{"spacing":{"blockGap":"","padding":{"right":"12%"}}}} -->
 <div class="wp-block-column" style="padding-right:12%;flex-basis:33%"><!-- wp:heading {"level":3} -->
-<h3 class="wp-block-heading">We recognize the crucial role architecture plays in shaping a sustainable future.</h3>
+<h3 class="wp-block-heading"><?php echo esc_html_x( 'We recognize the crucial role architecture plays in shaping a sustainable future.', 'Heading Text', 'twentytwentyfour' )?></h3>
```
Comment (MaggieCabrera) [inline patterns/columns.php:15]: why are you using `esc_html_x` in some strings and `esc_html__` in others?

Comment (devmuhib009) [reply patterns/columns.php:15]: I thought esc_html__ for small texts and esc_html_x for large paragraphs. Do I need to use esc_html_x for all stings? Please suggest me, I am new contributor.

Comment (huzaifaalmesbah) [reply patterns/columns.php:15]: if you use `esc_html_x` provide meaningful context for a better understanding. Context information for the translators.

**Comment by huzaifaalmesbah** [comment]
Pull Request #126 already Marge. I see you have the same request.

**Comment by devmuhib009** [comment]
What should I do now. Leave it as it is? @huzaifaalmesbah you know, I am new contributor. I did not noticed that one pull request is available. So, how can I close this PR?

**Comment by huzaifaalmesbah** [comment]
No problem. we need to close this PR. Thanks for your contribution.

**Pull request closed without merging**